src/Configuration/ConfigurationFactory.php

```diff
@@ -201,18 +207,18 @@ private static function retrieveExistingCoverageBasePath(
         ?string $existingCoveragePath,
         string $configDir,
         string $tmpDir
-    ): string {
```

Comment:
Does phpstan understand annotations like: ` @return array{0:bool,1:string}`?

Reply:
At least it doesn't seem to complain about it 👍